### ISSUE TITLE
Ranges for micfo and contina

### ISSUE THREAD

**Issue by jrivero** [client9/ipcat#34]
Hi!

Micfo have a very huge list of ip: https://ipinfo.io/AS53889#Blocks

And Contina is similar: https://ipinfo.io/AS63008#blocks

I'm thinking that maybe is a good idea maintain a database for identify by ASN

**Comment by client9** [comment]
Yeah its definitely the way to go.  Adding ASN to every entry will make it a lot easier to capture all the other IP blocks.

I'll take a look...

**Comment by jimdigriz** [comment]
Problem with ASN's is you get ISPs that put their users and the hosting under the same BGP advertisement.  Also, IP's do not get assigned to ASN's, they get *advertised* by ASN's so you will see a mix of suppliers and IP purposes per ASN also.

This is trivial to see if you take the IP's in your list, look up the ASN, then reverse the lookup to get a kinda superset of your datacenters.csv file.  You end up blacklisting BT, Virgin Media, etc (UK ISPs which a finger in every pie) and so on and so forth.
